__tests__/api.test.js

```diff
@@ -157,4 +157,68 @@ describe('GET /api/articles/:article_id/comments', () => {
             })
         })
         })
- 
\ No newline at end of file
+ 
+describe('POST /api/articles/:articles_id/comments', () => {
```

Comment:
As Ali said, what if we tried to post to articles/not-an-id ? It'd be great to see a test for that